### DISCUSSION TITLE
您好，这样是算是启动成功了吗？可是127.0.0.1:9096仍然无法访问

### DISCUSSION THREAD

**Discussion by JustStudyOne** [devlive-community/datacap#473] [Q&A]
![问题](https://github.com/devlive-community/datacap/assets/130358802/3c35d1cf-dab3-4436-86bd-850ab2ef4d65)

**Comment by qianmoQ** [comment]
@JustStudyOne 查看一下 logs 文件夹中的文件是否有错误信息输出？

**Reply by JustStudyOne** [reply]
[2023-11-10.log](https://github.com/devlive-community/datacap/files/13317690/2023-11-10.log)
有的，这是我的日志文件，但是我并不擅长java，可以请您看看吗？感谢！

**Reply by qianmoQ** [reply]
请使用 Java SDK 8 或者 11

**Comment by JustStudyOne** [comment]
好的，没问题

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by qianmoQ** [reply]
欢迎加入微信或者钉钉群，这样我们可以在第一时间内看到您反馈的问题

**Comment by JustStudyOne** [comment]
好的👌🏻，加过了

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
